phapp.yml

```diff
@@ -35,7 +35,6 @@ commands:
     drush locale:check && drush locale:update
   install: |
     chmod +w web/sites/default/settings.php &&
-    chmod +w web/sites/default/default.settings.php &&
```

Comment:
maybe it acutally should chmod +w the default folder, not the file? it's usually the folder which is set read only.